modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -1470,6 +1474,124 @@ public int getRootDiscussionMessagesCount(
 		return count;
 	}
 
+	@Override
+	public List<MBMessage> getSiteUserMessageBoardMessagesActivity(
+		long groupId, long userId, int start, int end) {
+
+		MBMessageTable aliasMBMessageTable1 = MBMessageTable.INSTANCE.as(
+			"aliasMBMessageTable1");
+
+		DSLQuery dslQuery = DSLQueryFactoryUtil.select(
+			MBMessageTable.INSTANCE
```

Comment:
Use mbMessageTable created before instead MBMessageTable.INSTANCE for all occurrences (when is possible).

Reply:
The pattern I found was this way in Liferay @joseabelenda